### PR TITLE
ISSUE-69 Deprecate apply operator

### PR DESCRIPTION
**Pull request by nakiostudio**
#69: Some fresh air for the library discontinuing the usage of the `<-` operator. For now the operator itself and other methods are marked as deprecated.

Apply operator `<-` replaced with:
```swift
myView.easy.layout(
  Width(200),
  Height(120)
)

view.easy.layout([
  Width(200),
  Height(240)
].when { isFirstItem })
```

The helper method `easy_clear()` is now:
```swift
view.easy.clear()
```

And also `easy_reload()` has changed:
```swift
view.easy.reload()
```

Thanks to @muukii for the ideas 🙌

### CODE REVIEW

File: EasyPeasy.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "EasyPeasy"
-  s.version          = "1.5.0"
+  s.version          = "1.5.1"
```
Comment (muukii) [inline EasyPeasy.podspec]: In this time, We have big changes deprecating operator (this was the key feature).
So, I think we need to update to `1.6.0` or `2.0.0`.
What do you think?

Comment (nakiostudio) [reply EasyPeasy.podspec]: Yup, I'm going to release `1.6.0` with the old methods marked as deprecated and then after sometime will release `2.0.0` forbidding the use of the deprecated ones.

Comment (muukii) [reply EasyPeasy.podspec]: @nakiostudio 
Okay!

**Comment by lipka** [comment]
I really liked how succinct the code was with the `<-` operator. Any specific reason for deprecating it?

However thumbs up on the `easy.reload()` and `easy.clear()` change, the underscore always felt out of place.

**Comment by nakiostudio** [comment]
Hi @lipka 👋 

I knew this wasn't going to satisfy every developer but I think it's a sensible change as the `<-` operator was one of the reasons for many devs to not to choose EasyPeasy. Also because after 3 years of Swift custom operators are in disuse for API designing.

However, you can stick to EasyPeasy `1.5.1` as deprecating those methods was the only change introduced on `1.6.0`. Another option is to drag [this file](https://gist.github.com/nakiostudio/543662179917eefe2db70cd6e1b81e5d) to your project to keep using the custom operator with future versions without dealing with deprecation warnings.

**Comment by nasht** [comment]
Hey @nakiostudio  - any reason why the extension you provide uses `<~` instead of the classic `<-` ?

**Comment by nasht** [comment]
Hey @nakiostudio. First off - thanks so much for building this fantastic lib. Our team are huge fans of the lib, and we've been using it on two very large projects here in Aus with pleasure. Sadly,  this 1.6.0 change means we need to eventually update thousands of usages of the previous syntax. One of the beautiful aspects of your lib was the concise and expressive nature of the `<- []` syntax. I hope you reconsider removing it, as certainly there is at least one team which loves the `<-`.  

Thanks again for the wonderful work! 👍

**Pull request merged**